api/admin/google_oauth_admin_authentication_provider.py

```diff
@@ -40,8 +40,8 @@ class GoogleOAuthAdminAuthenticationProvider(AdminAuthenticationProvider):
 
     SETTINGS = [
         { "key": ExternalIntegration.URL, "label": _("Authentication URI"), "default": "https://accounts.google.com/o/oauth2/auth" },
```

Comment:
This is also required, although if you don't provide it the default will be used. I saw some of the other ones had a default but were still required though.

Reply:
We should decide what to do for fields with defaults and be consistent about it, there are some that are required and some that aren't.

Reply:
Right, I don't know which ones should or should not be required but I will make this one required. I don't think select drop downs need a required attribute since a choice will _have_ to be made even if it's the default. But for this one it could be accidentally left blank.

Reply:
I think if a required field has a default, the default should show up in the text box and the 'Required' label should be shown. This makes it clear that you can't just wipe out the value in the text box.